<?xml version="1.0"?>

<launch>

  <node pkg="robot_explore" type="explore_server" respawn="false" name="explore_server" output="screen">
      <rosparam file="$(find robot_explore)/config/explore_costmap.yaml" command="load" ns="explore_costmap" />
  </node>

</launch>
